tests/regression/common/image.test.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+import path from 'path';
+import { LoginPage } from '../../../page-objects/login-page';
+
+test('file upload test', async ({ page }) => {
+  const loginPage = new LoginPage(page);
+  await loginPage.goToLoginPage();
+  await loginPage.login(process.env.LOGIN!, process.env.PASSWORD!);
+
+  await page.locator('.entry').first().click();
+  await page.waitForTimeout(1000);
+
+  await page.getByText('Hello TMS').click();
```

Comment:
Hmmm, what if there will e no "Hello TMS" record? Let's pretend you run tets on a new empty account?